app/javascript/packs/active_docs.js

```diff
@@ -0,0 +1,5 @@
+// We can define the 3scale plugins here and export the modified bundle
+import SwaggerUI from 'swagger-ui'
+import 'swagger-ui/dist/swagger-ui.css'
```

Comment:
Do we really want to inject css to HTML?
Can't we use external script so we can have CDN on it?

Reply:
it's not injected in the HTML, it's bundled inside a JS pack. As I commented before, we are aiming to stop providing modifying 3rd party assets from our CDN

Reply:
Ah if it is precompiled then we can use a CDN then

Reply:
The idea is to eliminate the future use of the CDN, stop providing third party modified assets.

Reply:
What are you talking about? Cloudfrount is a CDN as Akamai
If you provide compiled assets, then you can distribute them through a CDN
I am not talking about `cdn_assets` but about: injecting assets VS hosting them

injected assets can't be used by a CDN unlike compiled ones

Reply:
I meant **our** CDN.

Reply:
those packs are treated the same as the rest of Rails assets.

Reply:
Yes so out of topic :)
Closing